bundles/org.openhab.binding.chatgpt/src/main/feature/feature.xml

```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <features name="org.openhab.binding.chatgpt-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
 	<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+	<repository>mvn:org.openhab.ui.features/org.openhab.ui.features/${project.version}/xml/features</repository>
 
 	<feature name="openhab-binding-chatgpt" description="ChatGPT Binding" version="${project.version}">
 		<feature>openhab-runtime-base</feature>
+		<feature>openhab-ui-habot</feature>
```

Comment:
I don't think this is allowed for add-ons (and BTW core also isn't allowed to depend on ui/add-on features).
See my comment https://github.com/openhab/openhab-webui/issues/2995#issuecomment-3246912255.